requirements/common.txt

```diff
@@ -1,20 +1,21 @@
-django==1.8.2
-jsonfield==1.0.3
-requests==2.2.1
-django-cors-headers==1.1.0
-djangorestframework==3.1.2
-djangorestframework-jwt==1.6.0
-django-haystack==2.4.0
-django-allauth==0.21.0
-django-filter==0.11.0
-PyJWT==1.4.0
-django-braces
-django-autofixture
-django-ckeditor==5.0.2
-django-spaghetti-and-meatballs==0.1.1
-django-debug-toolbar==1.4
-coverage==4.0.3
-pytest-django==2.9.1
-django-import-export==0.4.2
-flake8
-XlsxWriter==0.9.3
+django==1.8.2

```

Comment:
You have changed the EndOfLine (eol) with CRFL endings instead of LF endings.
![image](https://user-images.githubusercontent.com/15031475/88916472-c40daa00-d283-11ea-9496-eca348d2a4a6.png)

Hence the entire files is reflected rewritten.
Do change these eol back to normal and update the specific dep. only!